gradle-module-plugin/src/main/kotlin/io/ia/sdk/gradle/modl/task/WriteModuleXml.kt

```diff
@@ -194,17 +194,12 @@ open class WriteModuleXml @Inject constructor(_objects: ObjectFactory) : Default
                     }
                 }
 
-                manifests().groupBy { it.scope }
-                    .forEach { (scope, manifests) ->
-                        manifests.flatMap { it.artifacts }
-                            .distinctBy { it.jarName }
-                            .forEach { artifact ->
-                                "jar" {
-                                    attribute("scope", scope)
-                                    -artifact.jarName
-                                }
-                            }
+                manifests().forEach { (jarName, scope) ->
+                    "jar" {
+                        attribute("scope", scope)
+                        -jarName
```

Comment:
This method was already metastasizing with transform logic prior to the current PRs so let's stop it doing that and just leave it to writing XML; pushing the transform upstream ...